Altis_Life.Altis/core/actions/fn_gutAnimal.sqf

```diff
@@ -20,7 +20,6 @@ switch (typeOf _animalCorpse) do {
     case "Goat_random_F": {_displayName = localize "STR_ANIM_Goat"; _item = "goat_raw";};
     case "Sheep_random_F": {_displayName = localize "STR_ANIM_Sheep"; _item = "sheep_raw";};
     case "Rabbit_F": {_displayName = localize "STR_ANIM_Rabbit"; _item = "rabbit_raw";};
-    default {_displayName = ""; _item = "";};
```

Comment:
I'm unsure why this was removed? If for some reason there is an animal that was not to be in the switch `_displayName` wouldn't have been defined